packages/contentful-migrations/core/0000000000006-configure-x-deploy-migration.js

```diff
@@ -82,7 +82,7 @@ module.exports = async function (migration, context) {
     headers: { 'X-Contentful-Content-Type': 'x-deploy' },
     data: {
       fields: {
-        name: {
+        internal_name: {
```

Comment:
we could also go for the latest contentful-migration helpers here:
https://github.com/jungvonmatt/contentful-migrations#migration-helpers
```
const { withHelpers } = require('@jungvonmatt/contentful-migrations');

module.exports = withHelpers(async (migration, context, helpers) => {
   const defaultLocale = await helpers.locale.getDefaultLocale();
   
   ....
   
});
```

Reply:
Using this is easier to use and as a benefit you also get intellisense/typescript support for `migration`, `context` && `helpers`

Reply:
Done